sdks/python/apache_beam/io/external/kafka.py

```diff
@@ -57,6 +58,10 @@
     ])
 
 
+def default_io_expansion_service():
+  return BeamJarExpansionService('sdks:java:io:expansion-service:shadowJar')
```

Comment:
Can we add some unit tests for BeamJarExpansionService and JavaJarExpansionService ?

Reply:
Sure, added some unit tests. This is also tested end-to-end in the cross language tests.